tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+
+We now have (almost) all relevant parameters to understand how Sentinel-1 views an area of interest. Resolution and pixel spacing are explained in more detail when we deal with speckle. In the next script, we'll highlight some practical aspects of what we just learned.
+
+```
+// The area of interest can be defined by drawing and area in the editor's map tab
+// We hardcode one here. 
+
+var geometry = ee.Geometry.Polygon(
+        [[[5.83, 52.74],
+          [5.83, 52.69],
+          [5.91, 52.69],
+          [5.91, 52.74]]], null, false);
+          
+// Get Sentinel-1 for an arbitrary 12 day period (i.e. a full orbit cycle)          
+var s1 = ee.ImageCollection('COPERNICUS/S1_GRD').
+  filterDate('2020-05-01', '2020-05-13').
+  filterMetadata('instrumentMode', 'equals', 'IW').
+  filterBounds(geometry);
+
+// Compose an ancillary property to categorize the images in this selection
+s1 = s1.map(function(f) {return f.set('platform_relorbit', 
+  ee.String(f.get('platform_number')).cat('_').
+  cat(ee.Number(f.get('relativeOrbitNumber_start')).format('%.0f')).
+  cat('_').cat(f.get('orbitProperties_pass')))});
+
+// Check which sensor/relative orbit combinations we have
+var orbits = ee.Dictionary(s1.aggregate_histogram('platform_relorbit'))
+
+print(orbits) // Check in console
+
+var keys = orbits.getInfo() // Needed to loop
+
+for (var k in keys) {
+  var color = 'blue'
+  if (k.charAt(0)==='B') {
+    color = 'green'
+  }
+  Map.addLayer(ee.Image().paint(ee.FeatureCollection(s1).filterMetadata('platform_relorbit', 'equals', k), 0, 1), {palette: [color]}, "Footprint" + k, false);
+}
+
+for (var k in keys) {
+  Map.addLayer(s1.filterMetadata('platform_relorbit', 'equals', k).first(), {bands: ['angle'], min: 30, max: 45 }, "Incidence angle: " + k, false);
+}
+
+Map.addLayer(ee.Image().paint(geometry, 0, 1), {palette: ['red']}, 'AOI', false);
+          
+```
+
+If you run the script *as is* you'll notice that the, relatively small, area of interest is where 2 DESC orbits overlap (with relative orbit numbers 37 and 110) AND also 2 ASC orbits (15 and 88). For both Sentinel-1A and -1B combined, that makes for 8 distinct coverages in a full orbit cycle. By displaying the individual footprints of each of the scenes, you should note that DESC scenes are indeed rotated slightly to the North-East, and ASC scenes to the North-West. 
+
+A curiosity is the situation for Sentinel-1B orbit 88. The scene boundary cuts right through the AOI. This is linked to the 25 seconds azimuth slice limits. Unfortunately, the azimuth slicing is not synchronized between Sentinel-1A and -1B (and may even drift somewhat over time). This means you need to compose the full AOI image cover by compositing two adjacent scenes in this case.
```

Comment:
The situation for Sentinel-1B orbit 88 is curious.
or
There is something curious about...

Reply:
revised